pkgs/applications/networking/opsdroid/default.nix

```diff
@@ -1,16 +1,34 @@
-{ lib, fetchFromGitHub, python3Packages }:
+{ lib, fetchpatch, fetchFromGitHub, python3Packages }:
 
 python3Packages.buildPythonPackage rec {
   pname = "opsdroid";
-  version = "0.25.0";
+  version = "0.28.0";
 
   src = fetchFromGitHub {
     owner = "opsdroid";
     repo = "opsdroid";
     rev = "v${version}";
-    sha256 = "0f32jf2rds9543akysxinf3hsgzr0w880xwcrcm1r2r0nhp8b8s5";
+    hash = "sha256-PG//UOSPNTAW6Xs8rSWWmnoBAODHmh6Js/iOes/XSAs=";
   };
 
+  patches = [
+    # https://github.com/opsdroid/opsdroid/pull/2018
+    # This patch makes opsdroid much more usable on NixOS.
+    (fetchpatch {
+      name = "support-static-dependency-environment.patch";
+      url = "https://github.com/opsdroid/opsdroid/pull/2018/commits/802e7f3500b935bae21ee915c17efa6f512ad4f0.patch";
+      hash = "sha256-ahE0FVgwxIM3HmF2WFChmBeyuPbhUyYyD/YPfcMmu9k=";
+    })
+    #
+    # This patch makes opsdroid functional on spec compliant homeservers.
+    # https://github.com/opsdroid/opsdroid/pull/2017
+    (fetchpatch {
+      name = "matrix-connector-spec-compliance.patch";
+      url = "https://github.com/opsdroid/opsdroid/pull/2017/commits/d2aae1f6648daf6fbd7b33370506f57df1be6f06.patch";
```

Comment:
this patch has been released in a new version. can you do the bump?

(note that fetching commits from pull requests is fragile: force-pushing the PR may break the link)

Reply:
I'm aware, but I'm not going to do the update until the governance situation in this community is fixed.